backend/development.ini.sample

```diff
@@ -289,7 +289,7 @@ email.notification.created_account.subject = [{website_title}] Someone created a
 email.notification.share_content_to_emitter.subject = [{website_title}] you shared {content_filename} to {nb_receivers} users
```

Comment:
Replace `you shared {content_filename} to {nb_receivers} users` by `You shared "{content_filename}" with {nb_receivers} users`

Reply:
done, french translation ?

Reply:
`Vous avez partagé "{content_filename}" avec {nb_receivers} utilisateurs`